.github/workflows/ci.yml

```diff
@@ -55,8 +55,18 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -U tox
+          pip install nbqa docutils
           sudo apt install -y graphviz pandoc
           pip install -e .
+
+          wget https://github.com/errata-ai/vale/releases/download/v2.23.0/vale_2.23.0_Linux_64-bit.tar.gz
+          mkdir $HOME/bin && tar -xf vale_2.23.0_Linux_64-bit.tar.gz -C $HOME/bin
+          export PATH=$HOME/bin:"$PATH"
+          echo "$HOME/bin" >> $GITHUB_PATH
+      - name: Lint documentation
+        run: |
+          echo "$PATH"
+          make docs-test
```

Comment:
I've gone through the vale documentation but is this the only way to download/install? Also I think the echo statements can be removed

Reply:
I'm installing like this to get the latest version. Once this is on the snap store, we can install via
```sh
sudo snap install vale --edge
```